tests/unittest_manager.py

```diff
@@ -131,7 +140,6 @@ def test_implicit_namespace_package(self) -> None:
     def test_namespace_package_pth_support(self) -> None:
         pth = "foogle_fax-0.12.5-py2.7-nspkg.pth"
         site.addpackage(resources.RESOURCE_PATH, pth, [])
-        pkg_resources._namespace_packages["foogle"] = []
 
```

Comment:
Let's not resolve the question above as I think it contains very valuable discussion for anybody revisiting this PR later on.

That said, are these tests still meaningful without the `_namespace_packages` injection?

Reply:
I think so. I'm not absolutely sure, but it looks like it's building an `ast` for those modules. My new tests don't do that. So I'd be reluctant to chop 'em.

Reply:
Yeah, even if they are not testing our `is_namespace` check anymore, they are checking if we can build these `asts` correctly. I guess that is a good test in itself.

Any regressions should probably be handled with a new test instead of expanding on these anyway.